lib/recipes/algebra-1/locales/es.yml

```diff
@@ -0,0 +1,3 @@
+es:
+  eoc:
+    chapter-review: Revisión del capítulo
```

Comment:
just curious why this doesn't match the english file

Reply:
Oh, oops, this was copy/paste and I forgot to remove it. It shouldn't be used right now since none of the books are in the collection as set to `es` language. It can be added again later if there's a Spanish version of the book.